Cargo.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 repository = "https://github.com/ruma/js_int"
 keywords = ["integer", "no_std"]
 categories = ["no-std"]
+rust-version = "1.46.0"
```

Comment:
NOTE: This feature wasn't supported in cargo for 1.46, but it still serves as a machine readable way to specify the MSRV, which is why I added it here.